routes/web.php

```diff
@@ -19,10 +20,31 @@
 
 Route::get('/login', [LoginController::class, 'show'])->name('login');
 Route::post('/login', [LoginController::class, 'login']);
+Route::get('/logout', [logoutController::class, 'logout']);
+
 
 Route::get('/home', [HomeController::class, 'index']);
 Route::get('/pagos', [HomeController::class, 'menu2']);
 Route::get('/caja', [HomeController::class, 'menu3']);
 Route::get('/reporteria', [HomeController::class, 'menu4']);
 
-Route::get('/logout', [logoutController::class, 'logout']);
+//resources
+Route::get('/degrees',[ resourcesController::class,'listDegrees']);
+Route::post('/newDegree',[ resourcesController::class,'createDegrees']);
+Route::post('/deleteDegree/{id}', [resourcesController::class, 'disableDegrees'])->name('deleteDegree');
+Route::post('/editDegree/{id}', [resourcesController::class, 'editDegrees'])->name('editDegree');
+Route::get('/trashDegrees', [resourcesController::class, 'trashDegrees'])->name('trashDegree');
+Route::post('/restoreDegree/{id}', [resourcesController::class, 'activeDegrees'])->name('enableDegree');
+
+
+Route::get('/sections',[ resourcesController::class,'listSections']);
+Route::post('/newSections',[ resourcesController::class,'createSections']);
+Route::post('/deleteSections/{id}', [resourcesController::class, 'disableSections'])->name('deleteSections');
+Route::post('/editSection/{id}', [resourcesController::class, 'editSection'])->name('editSection');
+
+Route::get('/courses',[ resourcesController::class,'listCourses']);
+Route::post('/newCourses',[ resourcesController::class,'createCourses']);
+Route::post('/deleteCourses/{id}', [resourcesController::class, 'disableCourses'])->name('deleteCourses');
+Route::post('/editCourses/{id}', [resourcesController::class, 'editCourses'])->name('editCourses');
```

Comment:
agrega las rutas a un grupo para que puedas manejar mejor el orden y los nombres de las rutas, example:

// Group routes for Degrees
```php
    Route::prefix('degrees')->group(function () { 
    Route::get('/', [resourcesController::class, 'listDegrees']);
    Route::post('/new', [resourcesController::class, 'createDegrees']);
    Route::post('/delete/{id}', [resourcesController::class, 'disableDegrees'])->name('deleteDegree');
    Route::post('/edit/{id}', [resourcesController::class, 'editDegrees'])->name('editDegree');
    Route::get('/trash', [resourcesController::class, 'trashDegrees'])->name('trashDegree');
    Route::post('/restore/{id}', [resourcesController::class, 'activeDegrees'])->name('enableDegree');
});
```

// Group routes for Sections
```php
    Route::prefix('sections')->group(function () {
    Route::get('/', [resourcesController::class, 'listSections']);
    Route::post('/new', [resourcesController::class, 'createSections']);
    Route::post('/delete/{id}', [resourcesController::class, 'disableSections'])->name('deleteSections');
    Route::post('/edit/{id}', [resourcesController::class, 'editSection'])->name('editSection');
});
```

Reply:
asi te quedaria la ruta para
listar: degress/ 
crear: degress/new
eliminar: degress/delete{id}
editar: degress/edit/{id}